<?xml version="1.0"?>
<launch>
    <arg name="bag_path" default="/home/gaobiao/Documents/201-2018/data/final_dwdx/final_2018-09-12-09-52-45_0_dwdx.bag"/>
    <arg name="rate" default="5"/>

    <!-- set a /global_example parameter -->
    <param name="file_prefix" value="$(arg bag_path)"/>

    <!-- play rosbag -->
    <node name="ros_data_replay" pkg="rosbag" type="play" output="screen" args="$(arg bag_path) --clock -r $(arg rate) " />

    <!-- convert velodyne_packets to velodyne_pointclo -->
    <!-- <include file="$(find rosbag_parser)/launch/nodelet_manager.launch" >
    </include>
    <include file="$(find rosbag_parser)/launch/cloud_nodelet.launch" /> -->

    <!-- <node pkg="rosbag_parser" name="velodyne_pointcloud_2_vdn" type="velodyne_pointcloud_2_vdn.py" output="screen"/> -->
    <!-- parse data -->
    <node pkg="rosbag_parser" name="video_saver" type="video_saver.py" output="screen" />
    <!-- <node pkg="rosbag_parser" name="velodyne_packets_saver" type="velodyne_packets_saver.py" output="screen" /> -->
    <!-- <node pkg="rosbag_parser" name="gps_saver" type="gps_saver.py" output="screen" /> -->

    <!-- <node pkg="rosbag_parser" name="ros_dwdx_saver" type="ros_dwdx_saver.py" output="screen" /> -->

</launch>
